compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/BXMLNSSymbol.java

```diff
@@ -35,6 +35,7 @@ public class BXMLNSSymbol extends BSymbol implements VariableSymbol {
      * Holds the namespace URI for lookup during semantic validations.
      */
     public String namespaceURI;
+    public Name compUnit;
```

Comment:
Add a comment saying it's applicable only for module XMLNS declaration. 

Maybe we should have a module XMLNS construct representation extending XMLNS, but we can look into this separately.

Reply:
Shall we add the comment here also?